Makefile

```diff
@@ -14,7 +14,7 @@ fclean: clean
 .PHONY: re
 re: clean all
 
-libft.a: libs/ft_string.c
+libft.a: libs/ft_string.c libs/ft_read_file.c
```

Comment:
We need to create another library (maybe `libbsq.a`) for separate concerns. e.g:

```make
libbsq.a: libs/bsq_input.c
        gcc -Wall -Wextra -Werror -Iheaders -c $^
        ar rc $@ bsq_input.o
        ranlib $@
```